File: programme/radian-to-degree/RadianToDegree.java
// Write a programme to convert Radian to Degree
import java.util.Scanner;

public class RadianToDegree {

  public static double radianToDegree(int radian) {
    return Math.round((radian * 180.0 / Math.PI) * 10000) / 10000.0;
  }

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    // taking radians input
    System.out.print("Enter the amount of Radians  : ");
    int radian = sc.nextInt();

    // printing the answer
    System.out.printf("\nAmount in Degrees = %f\n", radianToDegree(radian));

    sc.close();
  }
}

File: programme/radian-to-degree/radian-to-degree.cpp
// Write a programme to convert radians to degrees.
#include <iostream>

using namespace std;

int main()
{
    int radian;
    cout << "Radian  : ";
    cin >> radian;

    cout << "Degrees : " << radian * 180 / 3.141592653589793 << endl;

    return 0;
}
